python/ray/_private/utils.py

```diff
@@ -1896,3 +1896,32 @@ def update_envs(env_vars: Dict[str, str]):
             os.environ[key] = value.replace("${" + key + "}", os.environ.get(key, ""))
         else:
             os.environ[key] = value
+
+
+def parse_node_labels_json(
+    labels_json: str, cli_logger, cf, command_arg="--labels"
+) -> Dict[str, str]:
+    if not labels_json:
+        return {}
```

Comment:
The logical maybe should keep align with `parse_resources_json`, don't assign empty dict if the labels_json is null.

Reply:
No problem,  I has fixed.